test/Game.test.js

```diff
@@ -0,0 +1,18 @@
+import { GameBoard } from '../src/scripts/Game';
+
+describe('should check is a valide move', () => {
```

Comment:
- [ ] [OPTIONAL] Try to improve your descriptions so they really reflect what you are testing inside them. A good idea would be describe the function/module class in the describe first argument.